pyhive/common.py

```diff
@@ -237,13 +237,22 @@ def escape_string(self, item):
         # (i.e. only special character is single quote)
         return "'{}'".format(item.replace("'", "''"))
 
+    def escape_sequence(self, item):
+        l = []
+        for el in item:
+            v = self.escape_item(el)
+            l.append(v)
+        return "(" + ",".join(l) + ")"
```

Comment:
- loop can be condensed to `map(self.escape_item, item)`
- single quotes please
- the implementation of escape_number is currently just the identity function, so I believe this will error out if someone passes a list of numbers. Changing escape_number to return `str(item)` probably suffices.